language-server/src/server.js

```diff
@@ -31,10 +31,26 @@ import { addWorkspaceFolders, workspaceSchemas, removeWorkspaceFolders, watchWor
 import { getSemanticTokens } from "./semantic-tokens.js";
 import { JsonSchemaDocument } from "./json-schema-document.js";
 import * as Instance from "./json-instance.js";
+import picomatch from "picomatch";
 
 
 setShouldValidateSchema(false);
 
+let schemaFilePatterns = ["**/*.schema.json", "**/schema.json"];
+
+const isMatchedFile = (uri) => {
+  const matchers = schemaFilePatterns.map((pattern) => picomatch(
+    pattern,
+    { globstar: true,
+      matchBase: true,
+      dot: true,
+      nonegate: true }
+
+  ));
```

Comment:
The following is my preferred code style for cases like this. I'm surprised the eslint config didn't catch the object literal style and the empty line.

```suggestion
  const matchers = schemaFilePatterns.map((pattern) => {
    return picomatch(pattern, {
      globstar: true,
      matchBase: true,
      dot: true,
      nonegate: true
    });
  });
```